app.ts

```diff
@@ -801,11 +808,42 @@ app.get(
 	apisLimiter,
 	isAuthenticated,
 	async function (req, res) {
+		const sensorTypes = configManager.sensorTypes
+		const sensorGroups = configManager.namedScales
+
+		const scales: SensorTypeScale[] = []
+
+		for (const [key, group] of sensorGroups) {
+			for (const [, scale] of group) {
+				scales.push({
+					key: key,
+					sensor: group.name,
+					unit: scale.unit,
+					label: scale.label,
+					description: scale.description,
+					group: true,
```

Comment:
What do you need this flag for?

Reply:
Just to distinguish groups from the others